src/player.js

```diff
@@ -1811,10 +1811,12 @@ export default class Player extends FakeEventTarget {
         }
       });
       if (this.config.playback.playAdsWithMSE) {
-        this._eventManager.listen(this, AdEventType.AD_BREAK_START, () => {
-          const adData = this._adsController ? this._adsController.getAd() : null;
-          if (!!adData && adData.linear) {
-            this._detachMediaSource();
+        this._eventManager.listen(this, AdEventType.AD_LOADED, (event: FakeEvent) => {
```

Comment:
as I mention, multiple ads could be loaded but you want to activate the listener to the fit one. 
I think it'll be better to keep the pending ads as sorted array and take the first one in ad_break_start.

Reply:
1. this code move to kaltura player
2. `AD_LOADED` listening was the previous implement before @Yuvalke changed it to `AD_BREAK_START` in https://github.com/kaltura/playkit-js/pull/471 to fix a bug